package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@typescript-eslint/eslint-plugin": "^5.55.0",
     "@typescript-eslint/parser": "^5.55.0",
+    "axios": "^1.3.5",
     "eslint": "^8.36.0",
```

Comment:
`eslint` seems that can be moved to `devDependencies`

Reply:
true, will fix it